<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>241</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pb10">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>190</y>
     <width>88</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>10</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb50">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>220</y>
     <width>88</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>50</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb100">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>250</y>
     <width>88</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>100</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb500">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>280</y>
     <width>88</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>500</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbcoffee">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>190</y>
     <width>88</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>coffee(100)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbtea">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>220</y>
     <width>88</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>tea(150)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbmilk">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>250</y>
     <width>88</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>milk(200)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbReset">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>280</y>
     <width>88</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
